commit 1fb83666ae9a693d14de801184a04094a76ac885 (HEAD -> master, refs/changes/43/1043/2)
Author: Richard Kiene <richard.kiene@joyent.com>
Date:   2016-12-02T00:15:10+00:00 (2 years, 10 months ago)
    
    CMON-27 Make node-kstat build again
    Reviewed by: Bryan Cantrill <bryan@joyent.com>
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Bryan Cantrill <bryan@joyent.com>

diff --git a/package.json b/package.json
index a975901..443583f 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
 	"name":		"kstat",
-	"version":	"1.0.0",
+	"version":	"1.0.1",
 	"description":	"illumos libkstat bindings",
 	"homepage":	"https://github.com/joyent/node-kstat",
 	"author":	"Joyent (joyent.com)",
@@ -11,5 +11,8 @@
 	},
 	"devDependencies": {
 		"tape":		"4.6.2"
+	},
+	"scripts": {
+		"postinstall": "gmake $(eval echo ${MAKE_OVERRIDES})"
 	}
 }
